scripts/deploy-machine.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e #u
+shopt -s expand_aliases
+#set -o xtrace
+
+source scripts/testnet.sh
+source scripts/cleanup.sh
+
+# ---- On Deployment -----
+#
+# This is run on pushes to master, or release/**
+#
+# ENV Variables:
+# - GITHUB_REF: ie. `/refs/heads/mybranch`
+# - FUNDING_PRIV_KEY: funding private key, raw
+# - BS_PASSWORD: database password
+
+# Get version from package.json if not already set
+if [ -z "${RELEASE:-}" ]; then
+  RELEASE=$(node -p -e "require('./packages/hoprd/package.json').version")
+fi
+
+HOPRD_API_TOKEN='D3f4ul+!_2021'
+BS_PASSWORD='d3f4ul+!_2021'
+
+echo "Starting single preview pr node on xdai for hoprd:$RELEASE"
+start_testnet "pr-preview-xdai-$RELEASE" 1 "gcr.io/hoprassociation/hoprd:$RELEASE" "master-xdai" true
```

Comment:
We'd be funding each node on xdai for every PR that way. Shouldn't we use goerli right away instead?

Reply:
We should probably refactor the API so it can take the environment ID to know which provider to use.